src/drt/src/dr/FlexDR_maze.cpp

```diff
@@ -1551,8 +1558,8 @@ void FlexDRWorker::route_queue()
     gcWorker_->main();
     setMarkers(gcWorker_->getMarkers());
   }
-  if (debug) {
-    cout << "Starting with " << markers_.size() << " markers\n";
+  if (getDRIter() >= beginDebugIter) {
+    cout << "Starting worker " << getRouteBox() << " with " << markers_.size() << " markers\n";
```

Comment:
I'm not sure if it is a good idea to put all the debug info in the gui, since it can have large outputs. I think the gui could have cleaner version of the debug info.